packages/lesswrong/components/posts/AddToCalendar/AddToCalendarIcon.tsx

```diff
@@ -0,0 +1,67 @@
+import { registerComponent, Components } from '../../../lib/vulcan-lib';
+import React from 'react';
+import moment from '../../../lib/moment-timezone';
+import Add from '@material-ui/icons/Add';
+import AddToCalendar from '@culturehq/add-to-calendar';
+
+const styles = (theme: ThemeType): JssStyles => ({
+  plusIcon: {
+    position: 'absolute',
+    top: 11,
+    left: 11,
+    height: 10,
+    width: 10,
+    backgroundColor: 'white'
+  },
```

Comment:
This is not my favorite thing but I think it turned out ok. I couldn't find a MUI icon that I liked for this action, and the package that I used had their own calendar icon (which is not easily replaceable anyway), so I just overlaid a + on top of theirs to make it look more clickable.

Reply:
I don't think I can easily replace their icon, so it's probably not worth figuring it out unless we have a much better icon to replace it with